Policies, General Operations/Delegation of Website Duties.md

```diff
@@ -1,23 +0,0 @@
-### Delegation Of Website Duties to Logan L. Johnson
-
-This document hereby delegates all Website Duties to Logan L. Johnson (the Contributor), with Thomas Castleman (the Supervisor) as their direct supervisor within the CCT.
-
-#### Granted Powers
-These are the powers and capabilities that are henceforth granted to the Contributor in order to perform their duty to the best of their ability.
-
- * Full, administrative and management powers
- 
-#### Limitations
-These are the limitations imposed on the Contributor in the name of safety. These limitations do not indicate any lack of trustworthiness. Instead, they provide an opportunity to bring up larger issues with those who need to know about them.
-
- * Deletion of any data central to the functioning of Drauger OS or it's Community
- 	* Examples of this data include but are not limited to:
-      * The Drauger OS Wiki
-      * The Drauger OS Forums
-      * Drauger OS Email accounts
- 
-In addition to these limitations, it is requested that the Contributor keep track of major changes and issues and report them to the Supervisor, or directly to the CCT in the Supervisor's absence, at least once a quarter.
-
-#### Violation of Limitations
-
-In the event the Contributor violates their limitations, without direct, prior, written approval from either the Supervisor, the CCT, or the CPM (if Executive Action has been invoked), the Contributor submits to have all Granted Powers stripped from them. Furthermore, this Delegation is rendered null and void and is only preserved for record keeping purposes.
```

Comment:
Per the last sentence on this last line, this file must be kept for record keeping purposes. Find some way to mark as a voided document instead of deleting.

Reply:
Document has been re-added and a statement added that says the delegated authorities have been revoked.